mmv1/third_party/terraform/framework_utils/framework_config.go.erb

```diff
@@ -559,7 +559,7 @@ func GetCredentials(ctx context.Context, data ProviderModel, initialCredentialsO
 	}
 
 	if !data.AccessToken.IsNull() {
-		contents, _, err := pathOrContents(data.AccessToken.ValueString())
+		contents, _, err := transport_tpg.PathOrContents(data.AccessToken.ValueString())
```

Comment:
Does this function belong in the transport package?

Reply:
The package we are going to have are transport, acctest, tpgresource, verify, provider. I will move it to the verify package.